tsdb_k8s_queries/README.md

```diff
@@ -0,0 +1,89 @@
+## TSDB k8s query track
+
+This track main goal is keeping track of the performance of common k8s integration search requests.
+The queries, index templates and corpus data try to match production as close as possible.
+
+### The search requests
+
+The search requests of the following visualizations have been included in this track:
+* Average CPU usage per pod.
+* Average memory usage per pod.
+* Average cpu per container.
+* Average memory per container.
+* Average node cpu usage per container.
+* Average node memory usage per container.
+* Unique deployment count.
+* Percentile cpu usage per container.
+* Status per pod.
+* Transmitted network usage per pod.
+
+The search request is sourced from Kibana Lens in the k8s integration. For each of this search request a template is defined in operations.
+From each template three search requests are generated. First with last 15 minutes filter, then last 2 hours filter and finally with a last 24 hour interval. In case a `date_histogram` is used in the query template then each of this variation uses a different interval. A fixed interval of 30 seconds, 1 minute and 30 minutes repectively.
```

Comment:
```suggestion
From each template, three search requests are generated. First with the last 15 minutes filter, then the last 2 hours filter, and finally with a last 24-hour interval. In case a `date_histogram` is used in the query template then each of these variations uses a different interval. A fixed interval of 30 seconds, 1 minute, and 30 minutes respectively.
```